guix-science/packages/zotero.scm

```diff
@@ -0,0 +1,63 @@
+(define-module (guix-science packages zotero)
+  #:use-module (guix packages)
+  #:use-module ((guix licenses) #:prefix license:)
+  #:use-module (gnu packages base)
+  #:use-module (guix download)
+  #:use-module (nonguix build-system binary)
+  #:use-module (gnu packages gcc)
+  #:use-module (gnu packages gtk)
+  #:use-module (gnu packages xorg)
+  #:use-module (gnu packages glib)
+  #:use-module (ice-9 regex))
+
+    (define-public zotero (package
```

Comment:
The indentation is a bit adventurous :)  Please break the line after `zotero` and move the expression to column 0.